src/components/ContentFile.js

```diff
@@ -0,0 +1,54 @@
+import React from "react"
+import { DateTime } from "luxon"
+
+const transformDate = dateToTransform => {
```

Comment:
transforlDate to what ? it is not a good function name, try to explain what your function is doing in the name